ports/atmel-samd/boards/trellis_m4_express/pins.c

```diff
@@ -15,6 +15,9 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_OBJ_NEW_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_PB08) },
     { MP_OBJ_NEW_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_PB09) },
 
+    { MP_OBJ_NEW_QSTR(MP_QSTR_ACCELEROMETER_SDA), MP_ROM_PTR(&pin_PA12) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_ACCELEROMETER_SCL), MP_ROM_PTR(&pin_PA13) },
+
```

Comment:
Q: CPX has an `ACCELEROMETER_INTERRUPT` pin defined too. Is that connected on this board?

Reply:
nope just a secondary I2C port - since its an M4, its fast enough to query the chip for anything it needs

Reply:
OK!